[Feat] 메인페이지 카드 컴포넌트 2

mintnaka opened the pull request:
- 업적 카드 컴포넌트 구현 완료
- 마이페이지 전체 css 수정 보완
- 업적 카드에 들어가는 프로그레스바 공통 컴포넌트로 분리
- 컴포넌트 폴더 내 마이페이지 폴더에 스타일 폴더 생성해서 디렉토리 가독성 개선
- 마이페이지 컴포넌트들 props 수정




## PR 유형
- [x] 새로운 기능 추가
- [ ] 버그 수정
- [x] CSS 등 사용자 UI 디자인 변경
- [x] 코드에 영향을 주지 않는 변경사항(오타 수정, 탭 사이즈 변경, 변수명 변경)
- [x] 코드 리팩토링
- [x] 주석 추가 및 수정
- [ ] 문서 수정
- [ ] 테스트 추가, 테스트 리팩토링
- [ ] 빌드 부분 혹은 패키지 매니저 수정
- [x] 파일 혹은 폴더명 수정
- [ ] 파일 혹은 폴더 삭제

## PR 체크리스트
- [x] 커밋 메시지 컨벤션에 맞게 작성했습니다.
- [x] 변경 사항에 대한 테스트를 했습니다.(버그 수정/기능에 대한 테스트).

### PR 상세
![image](https://github.com/user-attachments/assets/e7e2395e-466e-4ab5-914f-abb8ee6b32e5)

마이페이지 구현 완료했습니다.

## 이슈
resolves #34

mintnaka committed:
[Feat] 메인페이지 카드 컴포넌트 2

- 업적 카드 컴포넌트 구현 완료
- 마이페이지 전체 css 수정 보완
- 업적 카드에 들어가는 프로그레스바 공통 컴포넌트로 분리
- 컴포넌트 폴더 내 마이페이지 폴더에 스타일 폴더 생성해서 디렉토리 가독성 개선
- 마이페이지 컴포넌트들 props 수정

#34

mintnaka committed:
Merge branch 'develop' of https://github.com/FRONTENDBOOTCAMP-12th/HobbyDobby-3rd into feature/mypage-card-components

somin2352 reviewed (changes requested):
props의 기본값 있을 경우, interface에 필수 사항 아님을 작성해주시면 좋을 것 같습니다(ex. level?:string)

mintnaka committed:
[Style/Design] 마이페이지 수정 요청 사항 반영

- 필수 props 기본 값 삭제
- 프로필 정보 부분 CSS 수정 : 닉네임을 좀 더 크게 강조
- 마이페이지 article 태그 클래스를 board 에서 article-container로 변경 : 리더보드의 board와 명칭 중복 방지

#34

mintnaka committed:
[Modify/Design] 마이페이지 수정 요청 사항 반영

- 필수 props 기본 값 삭제
- 프로필 정보 부분 CSS 수정 : 닉네임을 좀 더 크게 강조
- 마이페이지 article 태그 클래스를 board 에서 article-container로 변경 : 리더보드의 board와 명칭 중복 방지

mintnaka committed:
Merge branch 'feature/mypage-card-components' of https://github.com/FRONTENDBOOTCAMP-12th/HobbyDobby-3rd into feature/mypage-card-components

somin2352 reviewed (approved):
수정 사항 확인 완료했습니다!
